packages/core/examples/editableTextExample.tsx

```diff
@@ -78,15 +61,20 @@ export class EditableTextExample extends BaseExample<IEditableTextExampleState>
         return [
             [
                 <IntentSelect intent={this.state.intent} key="intent" onChange={this.handleIntentChange} />,
-                <label className={Classes.LABEL} key="maxlength">
-                    Max length
-                    <input
-                        className={Classes.INPUT}
+                <div className={Classes.FORM_GROUP} key="maxlength">
+                    <label className={Classes.LABEL} htmlFor={INPUT_ID}>Max length</label>
+                    <NumericInput
+                        id={INPUT_ID}
+                        className={Classes.FORM_CONTENT}
+                        min={0}
+                        max={300}
+                        onValueChange={this.handleMaxLengthChange}
                         placeholder="Unlimited"
-                        onChange={this.handleMaxLengthChange}
-                        value={this.state.maxLength !== undefined ? this.state.maxLength.toString() : ""}
+                        value={this.state.maxLength || ""}
                     />
-                </label>,
+                </div>,
+            ],
+            [
```

Comment:
oh dang i meant to remove the small text! cuz we added "(controlled)" to the placeholder